qiskit/schemas/qobj_schema.json

```diff
@@ -673,6 +700,70 @@
                         "name": {
                             "description": "Experiment name.",
                             "type": "string"
+                        },
+                        "n_qubits": {
+                            "description": "Number of backend qubits for converting back to OpenQasm registers",
+                            "type": "number"
+                        },
+                        "memory_slots": {
+                            "description": "Number of backend memory slots",
+                            "type": "number"
+                        },
+                        "clbit_labels": {
+                            "description": "A list of length memory_slots (from backend.configuration or in this header), where each element is either [creg_name, n] to identify the corresponding memory slot as the n-th element of creg creg_name, or null to identify that the corresponding memory slot has no corresponding creg",
+                            "items": [
+                                {
+                                    "items": [
+                                        {
+                                            "minLength": 1,
+                                            "title": "clreg",
+                                            "type": "string"
+                                        },
+                                        {
+                                            "description": "The register slot of the clreg",
+                                            "minimum": 0,
+                                            "title": "size",
+                                            "type": "integer"
+                                        }
+                                    ],
+                                    "maxItems": 2,
+                                    "minItems": 2,
+                                    "type": [
+                                        "array",
+                                        "null"
+                                    ]
+                                }
+                            ],
+                            "minItems": 1,
```

Comment:
Tests are failing as when qobj is serialized and there are no `clbit_labels` it adds an empty list to the Qobj. This causes `minItems` to not validate.